### ISSUE TITLE
Release QtPy 2.0.0

### ISSUE THREAD

**Issue by tlambert03** [spyder-ide/qtpy#300]
Hi folks, just checking in on the plans for releasing the qt6 support.  Not trying to rush you :) just couldn't find any issue/discussion.  Any last hurdles that need to be fixed that i could help with?

**Comment by dalthviz** [comment]
Hi @tlambert03 ! The idea we have is to do the release on December 22th (so next week). For the moment we are missing some issues regarding documentation (we will try to tackle them this week) and also check/review any new issues/PRs that could come around from people checking QtPy from master like for example https://github.com/spyder-ide/qtpy/pull/299 and https://github.com/spyder-ide/qtpy/pull/298

Just in case, the link to the milestone where we are tracking the missing things: https://github.com/spyder-ide/qtpy/milestone/29

Thank you so much for your interest and if you want to help us with anything of the above let us know! :)

**Comment by CAM-Gerlach** [comment]
Sorry, its my fault on the documentation delay; I'd meant to to it a week or two ago but I kept having to take care of other issues, and now I've been working 12+ hours a day, 7 days a week for the past week to finish a NASA project for by a deadline tomorrow. I just got it done at 5 am this morning, so I'll try to finish catching up on other things today (and food and sleep, lol) and then submit a PR on that tomorrow. Thanks for your patience!
